### PR TITLE
Base description of the project

### PR DESCRIPTION
**Pull request by Chroq**
## Description
Basic description of the project and cleaning of the readme.

## ToDo list:

- [x] Remove unused lines.
- [x] Add description
- [x] Add requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
-# FOSS Starter Kit
+# Hex
 
- ![Travis (.com)](https://img.shields.io/travis/com/marmelab/foss-starter-kit.svg) ![GitHub top language](https://img.shields.io/github/languages/top/marmelab/foss-starter-kit.svg) ![GitHub contributors](https://img.shields.io/github/contributors/marmelab/foss-starter-kit.svg) ![foss-starter-kit.svg](https://img.shields.io/github/license/marmelab/foss-starter-kit.svg) ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)
+This project implements a mobile and desktop version of the hex game.
```
Comment (Kmaschta) [inline README.md]: Desktop too?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
 
-During a major spring cleanup of Marmelab open-source projects, we spent a lot of time homogenizing access to these projects. This repository is intended to centralize the sample files to be added to our future open-source projects.
+## Requirements
+
+Actually, you need to have this tools installed to code in the project.
```
Comment (Kmaschta) [inline README.md]: Actually, you can remove this sentence

File: README.md (hunk 1 above)
Comment (Chroq) [reply README.md]: Think so, no ? Not a priority but must be playable in desktop, I believe.

File: README.md (hunk 2 above)
Comment (Chroq) [reply README.md]: Done ;) TY

**Pull request merged**